code/hdb/hdbstandard.q

```diff
@@ -1,7 +1,26 @@
 // reload function
 reload:{
+	
+	if[.z.w in key .hdb.reloadcalls;
+		.hdb.reloadcalls[.z.w]:1b;
+		.lg.o[`reload;"reload call received from handle ", string[.z.w], "; reload calls pending from handles ", ", "sv string where not .hdb.reloadcalls];
+		if[not all .hdb.reloadcalls;:(::)]];
```

Comment:
```suggestion
		if[not all .hdb.reloadcalls;:(::)];
	];
```

IMO the closing `]` of a multi-line `if` block should be on it's own line and aligned with the opening `if`